src/components/language-selector/language-modal/LanguageModal.js

```diff
@@ -28,6 +28,10 @@ const LanguageModal = ({ t, tReady, onLeave, link, className, ...props }) => {
             </button>
           )}
         </div>
+        <p className='lh-copy charcoal f6'>
+          {t('languageModal.translationProjectIntro')}<br/>
+          <a href="https://www.transifex.com/ipfs/public/" rel="noopener noreferrer" target="_blank" class="link blue">{t('languageModal.translationProjectLink')}</a>
+        </p>
```

Comment:
Feel free to tweak style / text so it looks better :pray: initial PoC looks like this:
![2020-05-28--00-32-10](https://user-images.githubusercontent.com/157609/83079026-34792e80-a07b-11ea-903a-894d9a888c79.png)